llvm/test/Transforms/InstCombine/overflow_to_sat.ll

```diff
@@ -13,6 +13,18 @@ define i32 @uadd(i32 %x, i32 %y) {
   ret i32 %s
 }
 
+define i32 @uadd_cumm(i32 %x, i32 %y) {
```

Comment:
Is `cumm` here supposed to be short for `commutative`? If so it should be `comm`